src/main/java/object/user/User.java

```diff
@@ -0,0 +1,75 @@
+package object.user;
+
+import com.github.javafaker.Faker;
+
+
+import java.util.Objects;
+
+public class User {
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+
+    public User(String passwordUser) {
+        this.firstName = new Faker().name().firstName();
+        this.lastName = new Faker().name().lastName();
+        this.email = new Faker().internet().emailAddress();
+        if (passwordUser.equals("valid")) {
+            this.password = new Faker().internet().password(8, 10, true, true);
+        } else if (passwordUser.equals("invalid")) {
+            this.password = new Faker().internet().password(1, 2, false);
+        }
```

Comment:
Shouldn't be in constructor. If you want you can create method something like createRandom() and move logic there. Constructor should be more about variable initialization.

Reply:
fixed